File: Swexpertacademy/cpp/최대구간합.cpp
#include <iostream>
#include <fstream>
int arr[1000001];
int main(){
	freopen("input.txt", "r", stdin);
	
	ios_base :: sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);

	int T;
	cin>>T;
	for(int tc=1; tc<=T; tc++){
		int N;
		cin>>N;
		 
		for(int i=0; i<N; i++) {
			cin>>arr[i];
		}
	}
	return 0;
}

File: Swexpertacademy/python/최대구간합.py
import sys
sys.stdin = open('input.txt')

for tc in range(1, int(input())+1):
    arr = list(map(int, input().split()))
    maxs = [0]*arr[0]
    mins = [0]*arr[0]
    arr.pop(0)
    ans = sum(arr)
    temp = 0
    tag = False
    for a in arr:


    print(ans)
